src/collection.js

```diff
@@ -314,7 +314,14 @@ export default class Collection {
     }
     return this.db.create(newRecord).then(record => {
       return {data: record, permissions: {}};
-    });
+    })
+      .catch(err => {
+        if (options.useRecordId) {
```

Comment:
That could be another check to add, yes; though we may want to allow this in tests, so I'm not sure - after all, should we forbid creating virtually deleted records?